### PR TITLE
[13.0][MIG] pos_cash_move_reason

### PR DESCRIPTION
**Pull request by acsonefho**
Migration of module pos_cash_move_reason from `12.0` to `13.0`

### CODE REVIEW

File: pos_cash_move_reason/models/pos_move_reason.py [hunk 1]
```diff
@@ -12,9 +12,7 @@ class PosMoveReason(models.Model):
     @api.model
     def _default_journal_ids(self):
         AccountJournal = self.env["account.journal"]
-        journals = AccountJournal.search(
-            [("journal_user", "=", True), ("type", "=", "cash")]
-        )
+        journals = AccountJournal.search([("type", "=", "cash")])
```
Comment (legalsylvain) [inline pos_cash_move_reason/models/pos_move_reason.py:15]: why removing the ``journal_user`` constraint ?

**Review by legalsylvain — COMMENTED** [review]
one minor remark. Otherwise LGTM. 
(no tested)

File: pos_cash_move_reason/models/pos_move_reason.py (hunk 1 above)
Comment (acsonefho) [reply pos_cash_move_reason/models/pos_move_reason.py:15]: This field doesn't exist in v`13.0`

**Review by Cedric-Pigeon — APPROVED** [review]
LGTM

File: pos_cash_move_reason/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2016 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "POS cash in-out reason",
+    "summary": """""",
```
Comment (ivantodorovich) [inline pos_cash_move_reason/__manifest__.py:5]:
```suggestion
    "summary": "Define reasons for "Put Money In" or "Take Money Out",
```

File: pos_cash_move_reason/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "author": "ACSONE SA/NV," "GRAP," "Odoo Community Association (OCA)",
```
Comment (ivantodorovich) [inline pos_cash_move_reason/__manifest__.py:6]:
```suggestion
    "author": "ACSONE SA/NV, GRAP, Odoo Community Association (OCA)",
```

File: pos_cash_move_reason/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    "website": "https://www.github.com/OCA/pos",
+    "category": "Point Of sale",
+    "version": "13.0.3.0.2",
```
Comment (ivantodorovich) [inline pos_cash_move_reason/__manifest__.py:9]:
```suggestion
    "version": "13.0.1.0.0",
```

**Review by ivantodorovich — CHANGES_REQUESTED** [review]

**Pull request closed without merging**